TerminalGuiFluentTesting/GuiTestContext.cs

```diff
@@ -550,4 +562,25 @@ private void SendWindowsKey (ConsoleKeyMapping.VK specialKey)
 
         WaitIteration ();
     }
+
+    /// <summary>
+    /// Sets the input focus to the given <see cref="View"/>.
+    /// Throws <see cref="ArgumentException"/> if focus did not change due to system
+    /// constraints e.g. <paramref name="toFocus"/>
+    /// <see cref="View.CanFocus"/> is <see langword="false"/>
+    /// </summary>
+    /// <param name="toFocus"></param>
+    /// <returns></returns>
+    /// <exception cref="ArgumentException"></exception>
+    public GuiTestContext Focus (View toFocus)
```

Comment:
Is there some way to not have to re-implement EVERY action on `Application` in `GuiTestContext`? 

You added `Focus`. I added `SendKey`. Next we'll need more mouse, navigation, etc... methods....

Reply:
Hmn... its a good question.  I am not sure.

I wanted to do 'whole system testing' so the methods like Down etc actually synthesise the net/windows raw inputs - this exercises the full input processor pipe etc.

SendKey steps away from that but I think thats sensible i.e. just practical.

I will have a think but I can't think of an immediate way to expose e.g. all members of Application.